### ISSUE TITLE
Incorrect USER examples

### ISSUE THREAD

**Issue by rcr** [ircdocs/modern-irc#56]
I believe both of these examples are incorrect:

```
USER guest tolmoon tolsun :Ronnie Reagan
                              ; No ident server
                              ; User gets registered with username
                              "~guest" and real name "Ronnie Reagan"

USER guest tolmoon tolsun :Ronnie Reagan
                              ; Ident server gets contacted and
                              returns the name "danp"
                              ; User gets registered with username
                              "danp" and real name "Ronnie Reagan"
```

**Comment by DanielOaks** [comment]
Huh, good point! I'll update both those, thanks a bunch for pointing out the issue

**Comment by DanielOaks** [comment]
Fixed with c2ff38245e19d18f58187062592a26ff8347ca25
